src/cloud/docker/docker-mode-developer.md

```diff
@@ -16,33 +18,15 @@ Large files (>1 GB) can cause a period of inactivity. DB dumps and archive files
 
 {%include cloud/note-docker-config-reference-link.md%}
 
-{:.procedure}
-To launch the Docker environment in developer mode:
-
-1. Download a Magento application template from the [Magento Cloud repository][cloud-repo] and copy the files to your {{site.data.var.mcd-prod }} project directory. Be careful to select the branch that corresponds with the Magento version.
-
-1. Add your [Magento access credentials][magento-creds] to the `auth.json` file.
-
-1. Run the installation script, `init-docker.sh` to install template dependencies and add the default hostname to your `/etc/hosts` file.
-
-   ```bash
-   curl https://raw.githubusercontent.com/magento/magento-cloud-docker/<magento-cloud-docker-package-version>/bin/init-docker.sh | bash
-   ```
-
-   For `<package-version>`, use the [latest release of the {{site.data.var.mcd-package}}].
+## Prerequisites
```

Comment:
H2 seems odd for prerequisites because then the procedure becomes a subset. I suppose Bolding is not preferred style for the linter...but not sure what is. Maybe procedure class again...or maybe a `>` like you used in the synchronizing data topic? Just thinking on ideas... 🤔

Reply:
Fixed locally.